Child module org.eclipse.ec4j does not exist

```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Child module /home/ppalaga/orgs/tools/ec4e/org.eclipse.ec4j of /home/ppalaga/orgs/tools/ec4e/pom.xml does not exist @ 
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.eclipse.ec4e:org.eclipse.ec4e.parent:0.1.0-SNAPSHOT (/home/ppalaga/orgs/tools/ec4e/pom.xml) has 1 error
[ERROR]     Child module /home/ppalaga/orgs/tools/ec4e/org.eclipse.ec4j of /home/ppalaga/orgs/tools/ec4e/pom.xml does not exist
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

A PR follows.

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,6 @@
 	<version>0.1.0-SNAPSHOT</version>
 	<packaging>pom</packaging>
 	<modules>
-        <module>org.eclipse.ec4j</module>
 		<module>org.eclipse.ec4e</module>
 		<module>org.eclipse.ec4e.repository</module>
 	</modules>

